Personal.md (#2)
spinningrachel/cycode: docs/personal.md

> 

Reviewer: Enter an explanation how to navigate to this area.

## Personal details

In your personal details, you can:

- see the name and email address used in your account

- change the name in your account

## Change password

> To change your password, enter your current password and then enter a new one.

Author: Is an authenticator app necessary to change passwords also? Or just for MFA (in which case the prerequisite would be in that section)?

> To change your password, enter your current password and then enter a new one.

Author: @spinningrachel ?

## Multi-factor authentication

Enable multi-factor authentication for an added level of security. 

#### To enable multi-factor authentication:

> 1. Use your phone to scan the QR code that appears on the screen.

Reviewer: ```suggestion
1. From the Personal Settings area, in the My Profile pane click MFA. 
    
    The Multi-Factor Authentication window loads, similar to the following: 
    
    <insert mfa.png (I already added it to the folder)>
    
2. Toggle the button to enable MFA.

```

	You need to scan the code using an authenticator app like Google Authenticator. 

> 2. Enter the 6-digit code from your authenticator app.

Reviewer: ```suggestion
2. From Cycode, enter the 6-digit code.
```

> 3. Save your recovery code.

Reviewer: ```suggestion
3. Save your recovery code in a safe place.
```

> 	If you lose your phone, the recovery code enables you to log in.

Reviewer: ```suggestion
	The recovery code enables you to log in if you lose your phone, 
```

## Access tokens

You automatically have the option to integrate everything shown in [**Integrations**](link to integrations page). If you need to integrate something else, create a personal access token. With these tokens, you authorize other applications to integrate Cycode.  

#### To create a personal access token:

> 1. Click **Create New**.

Reviewer: @rgottesman try adding results and images to the rest of the processes

Author: The personal access token page looks different when there aren't any tokens created yet. Can I delete the token you created, or should I wait until we get a clean account?
